piwheels/master/the_scribe.py

```diff
@@ -349,6 +361,76 @@ def write_project_page(self, package):
         except FileExistsError:
             pass
 
+    def delete_package(self, package):
+        """
+        Attempts to remove the index and project page directories (including all
+        known wheel files) of the specified *package*.
+
+        :param str package:
+            The name of the package to delete.
+        """
+        if len(package) == 0:
+            # refuse to delete /simple/ and /project/ by accident
+            self.logger.error('not deleting - package name is empty')
+            return
+
+        files = {f for f in self.db.get_package_files(package)}
+
+        for name in ('simple', 'project'):
+            self.logger.info('deleting /%s/ dir for %s', name, package)
+            pkg_dir = self.output_path / 'simple' / package
+            canon_dir = pkg_dir.with_name(canonicalize_name(pkg_dir.name))
+
+            for dir in (pkg_dir, canon_dir):
```

Comment:
Nitpick: `(pkg_dir, canon_dir)` should be `{pkg_dir, canon_dir}` - eliminates the second go round the loop if the name is canonical anyway.

Reply:
Nice